package.json

```diff
@@ -42,12 +52,13 @@
     "eslint-config-airbnb-base": "13.1.0",
     "eslint-plugin-import": "2.14",
     "jest": "23.6.0",
+    "lodash.filter": "4.6.0",
     "uuid": "3.3.2"
   },
   "peerDependencies": {
     "aws-sdk": ">= 2.x.x",
     "axios": ">= 0.x.x",
-    "lodash": ">= 4.x.x",
+    "lodash.filter": ">= 4.x.x",
```

Comment:
I believe we should add all lodash packages there as well

Also, to make sure it works, add those packages in the "example", using various 4.X versions to make sure it works when someone uses different lodash package versions

Also, should we add other packages like bluebird, chalk, etc? 

I'd like the example to be the most complete possible, so let's see if there is a compatibility issue when defining `dependencies` in the package.json example that mismatch the plugin dependencies

Reply:
No need lodash here